storages/python/python-storage-plugins/README.md

```diff
@@ -1,24 +1,58 @@
 Python storage plugins distributed with DLite
 =============================================
-This directory contains additional storage plugins written in Python,
-including
+This directory contains additional storage plugins written in Python.
 
-* blob - a plugin for Binary Large OBjects (BLOBs).
 
-* bson - a plugin for BSON (binary JSON). Requires the pymongo package.
+Generic plugins
+---------------
+Generic plugins supports all types of DLite instances (including data
+instances, metadata, collections, etc).
 
-* csv - a plugin for CSV files.
+- **bson**: Plugin for BSON (binary JSON). Requires the pymongo package.
+  Require: bson
```

Comment:
```suggestion
- **bson**: Plugin for BSON (binary JSON). 
  Requires: bson, pymongo
```

Reply:
The bson plugin only depends on the [bson](https://pypi.org/project/bson/) PyPI package, that doesn't depend on pymongo.